### ISSUE TITLE
Beware the 2% ad sharing

### ISSUE THREAD

**Issue by becvert** [cranberrygame/cordova-plugin-ad-admob#18]
Why not advertising more clearly the ad sharing?
In it's own License section for instance

> //if no license key, 2% ad traffic share for dev support.
>     //you can get paid license key: https://cranberrygame.github.io/request_cordova_ad_plugin_paid_license_key
>     //window.admob.setLicenseKey("yourEmailId@yourEmaildDamin.com", "yourLicenseKey");

**Comment by renandecarlo** [comment]
This is open-source, I think you're free to fork and remove the 2% if you are willing to.

I also agree that there should be something clearer.

**Comment by becvert** [comment]
Hi @renandecarlo 

open source or not, this is not the point for me.
there are quite a few admob plugins out there. none are very upfront.
I'm pretty sure a lot of devs don't realise that there is the ad sharing.

interesting points there too [https://github.com/floatinghotpot/cordova-admob-pro/issues/450](https://github.com/floatinghotpot/cordova-admob-pro/issues/450)

I switched over to [cordova-plugin-admob-free](https://www.npmjs.com/package/cordova-plugin-admob-free) since then.

thanks

**Comment by renandecarlo** [comment]
I'm on par with that 2%+ conversation. That is actually one of the reasons I came here, besides some other problems I was having with admob-pro.
I'm ok with 2%, but I can't trust it if it has to retrieve and send information from a 3rd party server. I wasn't even aware of that before that conversation.

That "crackers" story of the 30% didn't convince me.

I think most of the admob plugins try to earn money by kinda hiding that there is a commission.

The problem with cordova-plugin-admob-free is that there are no mediations.

**Comment by becvert** [comment]
Clearly some plugins are outright scams
and admob-pro as some big issues.

This is to us to put effort into adding mediation plugins
That's how works the community.
I started to add [Facebook Audience mediation](https://github.com/becvert/cordova-plugin-admob-mediation-facebook) (not on npm yet)
It's quite straight forward.

**Comment by code4youreal** [comment]
I do not care the 2% but this plugin is unusable on iOS If I build with cordova-plugin-ad-admob then I always get build failed message, and the support is a joke! Someone told me this plugin do not support iOS 10 so I do not recommend.

**Comment by rahianeja** [comment]
I agree not informing about commission upfront. 2% is okay as long as it is clear its 2%. I tried to read plugin files, but couldn't figure out how this commission is going and where is the logic. I also feel this is only to get devs pay for license.

**Comment by lpopova** [comment]
If you don't call prepare and show any ads, is there some communication between the plugin and the admob? Am I safe from GDPR if I just don't call ads functions but its still in the apk? 
For users that don't want tracking is not showing the ads (and preloading them) enough?
